clang/include/clang/Driver/Options.td

```diff
@@ -7172,6 +7174,9 @@ let Visibility = [CC1Option, CC1AsOption] in {
 def debug_info_macro : Flag<["-"], "debug-info-macro">,
   HelpText<"Emit macro debug information">,
   MarshallingInfoFlag<CodeGenOpts<"MacroDebugInfo">>;
+def no_system_debug : Flag<["-"], "no-system-debug">,
+  HelpText<"Do not emit debug information for declarations in system headers">,
+  MarshallingInfoFlag<CodeGenOpts<"NoSystemDebug">>;
```

Comment:
Why add a new option here?  We should be able to use the old `-fsystem-debug` syntax for both command line and `-cc1` usage.

Reply:
Removed -no-system-debug.